### DISCUSSION TITLE
Telegram Bot 快捷发送消息到Flomo

### DISCUSSION THREAD

**Discussion by puzige** [flomoapp/3rd-party-tools#78] [分享创造]
目前仅支持发送文字，还在持续开发中

🔗  https://t.me/flomo_note_bot

![image](https://github.com/flomoapp/3rd-party-tools/assets/32303768/4bb2f504-8938-46c7-b5a2-c5b7d7f17272)

**Comment by Duonos** [comment]
是不是已经停用了？发送信息触发不了任何回复。
